### DISCUSSION TITLE
Interfaces Usage | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1687] [Q&A]
# Interfaces Usage | Seeed Studio Wiki

Hardware and Interfaces Usage for J401 carrier board

https://wiki.seeedstudio.com/J401_carrierboard_Hardware_Interfaces_Usage/

**Comment by bohare** [comment]
Hi, is there a specific cable or connector which I should use to connect to the CAN interface of the J401 board for data transfer from an external source?
Thank you!
